### ISSUE TITLE
[request] column text edition mode

### ISSUE THREAD

**Issue by atesin** [pkulchenko/ZeroBraneStudio#1170]
you know, column edition mode similar to that present in ultraedit and notepad++ with `Alt` key

would be useful in certain situations like editing multiline comment blocks, text diagrams, manual identation, etc

**Comment by atesin** [comment]
nevermind ....  i discovered this ACCIDENTALLY, by holding `Shift` + `Alt` + mouse drag

please add this trick in [tips and tricks](https://studio.zerobrane.com/doc-tips-and-tricks) to be available for everyone

**Comment by pkulchenko** [comment]
Yes, it's already available. I don't mind updating tips and tricks page, but it's already added to the editor shortcuts page (https://studio.zerobrane.com/doc-editor-keyboard-shortcuts) under the navigation and selection sub-header. Is it worth still adding to the tips&tricks page?

**Comment by atesin** [comment]
being like that, idk, there are many more shortcuts that i didn't know it, i am very new to zerobrane and lua programming

but a general "which are the editor keyboard shortcuts" entry may be a good idea ... but however yes, i think it should be **also** in a separate "tips and tricks" entry because is a whole different edition mode
